pkg/util/utils_test.go

```diff
@@ -22,6 +22,28 @@ import (
 	"github.com/blang/semver"
 )
 
+func TestValidateString(t *testing.T) {
+	testData := []struct {
```

Comment:
I dont undrestand why the test has to count the length of the valid options,
the test should be called, no need for doing this.